tests/models/nn/test_transformer_data_preparator.py

```diff
@@ -59,3 +66,186 @@ def test_from_interactions(
         )
         assert len(actual.weights) == len(expected_weights)
         assert all(actual_list == expected_list for actual_list, expected_list in zip(actual.weights, expected_weights))
+
+
+class TestSessionEncoderDataPreparatorBase:
+
+    def setup_method(self) -> None:
+        self._seed_everything()
+
+    def _seed_everything(self) -> None:
```

Comment:
I don't think that we need it for data preparator. All is determentistic as far as I see.